cmd/gcrane/cmd/gc.go

```diff
@@ -29,28 +31,57 @@ func init() { Root.AddCommand(NewCmdGc()) }
 // NewCmdGc creates a new cobra.Command for the gc subcommand.
 func NewCmdGc() *cobra.Command {
 	recursive := false
+	before := int64(-1)
+	pattern := ""
+
 	cmd := &cobra.Command{
 		Use:   "gc",
 		Short: "List images that are not tagged",
 		Args:  cobra.ExactArgs(1),
 		Run: func(_ *cobra.Command, args []string) {
-			gc(args[0], recursive)
+			gc(args[0], recursive, time.Unix(before, 0), pattern)
 		},
 	}
 
 	cmd.Flags().BoolVarP(&recursive, "recursive", "r", false, "Whether to recurse through repos")
+	cmd.Flags().Int64VarP(&before, "before", "b", time.Now().Unix(), "Oldest upload time (unix timestamp)")
```

Comment:
If we support this I'd love to do it in such a way that users don't have to handle Unix timestamps. Maybe an relative age instead of a date? --before=30d perhaps?

Reply:
Agree. Sounds like a duration flag. The problem with duration flag tho is that it does only accept ["ns", "us" (or "µs"), "ms", "s", "m", "h"](https://golang.org/pkg/time/#ParseDuration) (and this seems to be a [touchy subject](https://github.com/golang/go/issues/11473) in the golang community). Accepting only day would be a bit limiting too. Do we go for a string flag with a custom parser or do we keep the `Duration` limitations?

Reply:
How would you feel about using cobra's [`Duration` flag](https://godoc.org/github.com/spf13/pflag#Duration) for now, which would only support up to hours, and when/if someone complains we can evaluate custom parsing based on need. 

Worst case, `--before=168h` is still better I think than whatever shell expression would provide the equivalent Unix timestamp, even though `--before=7d` would be even more idea.

Reply:
Agree. Changes incoming.